apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/model/BrevModel.kt

```diff
@@ -18,24 +19,45 @@ enum class Status {
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 data class Adresse(
-    val navn: String? = null,
-    val adresse: String? = null,
+    val adresseType: String,
+    val adresselinje1: String? = null,
+    val adresselinje2: String? = null,
+    val adresselinje3: String? = null,
     val postnummer: String? = null,
     val poststed: String? = null,
-    val land: String? = null
+    val landkode: String,
```

Comment:
Slår meg som fristande med ein type for land og landkode

Reply:
Type enum, eller bare en klasse som sikrer et visst mønster?